switools/swisigningservice.py

```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3.7
```

Comment:
Can this be `python3`?

Reply:
I guess (it is not used, since that binary is egg-i-fied).